client/src/store/reducer/errorsBagReducer.js

```diff
@@ -1,12 +1,38 @@
 const initialState = {};
 
+const swapKeyValuesInObject = (obj, key1, key2) => {
+  const objectToSwapValues = {};
+  [objectToSwapValues[key1], objectToSwapValues[key2]] = [
+    objectToSwapValues[key2],
+    objectToSwapValues[key1],
```

Comment:
I believe dot notation would look better here!

Reply:
Dot notation won't work here right? if theyre variables that are passed in

Reply:
key1,key2 are strings. Would only work this way